### PR TITLE
chore: Make app icons consistent Material Symbols with same settings

### PR DESCRIPTION
**Pull request by susnux**
## Summary

Make all app icons the same style Material Symbols (e.g. files was still MDI, and some had different optical size).

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by nickvergessen — APPROVED** [review]

**Review by jancborchardt — APPROVED** [review]
Looks very nice!

File: apps/files_external/img/app-dark.svg
Comment (AndyScherzinger) [inline apps/files_external/img/app-dark.svg:1]: not sure about changing this icon this way. I'd rather stick to the former one which is also Material Symbol https://fonts.google.com/icons?icon.query=open&selected=Material+Symbols+Outlined:open_in_new:FILL@1;wght@400;GRAD@0;opsz@24&icon.size=24&icon.color=%23e8eaed

What do you think @jancborchardt

File: apps/files_external/img/app.svg
Comment (AndyScherzinger) [inline apps/files_external/img/app.svg:1]: same as here https://fonts.google.com/icons?icon.query=open&selected=Material+Symbols+Outlined:open_in_new:FILL@1;wght@400;GRAD@0;opsz@24&icon.size=24&icon.color=%23e8eaed

File: apps/files_versions/img/app-dark.svg
Comment (AndyScherzinger) [inline apps/files_versions/img/app-dark.svg:1]: https://fonts.google.com/icons?icon.query=restore&selected=Material+Symbols+Outlined:settings_backup_restore:FILL@1;wght@400;GRAD@0;opsz@24&icon.size=24&icon.color=%23e8eaed the former icon has also been used as the icon on the tab. So for consistency I suggest to keep that one or replace the tab icon as well in the right sidebar

File: apps/files_versions/img/app.svg
Comment (AndyScherzinger) [inline apps/files_versions/img/app.svg:1]: same as here https://github.com/nextcloud/server/pull/46391/files#r1672231666

**Review by AndyScherzinger — APPROVED** [review]
some minor changes suggested

File: apps/files_external/img/app-dark.svg
Comment (susnux) [reply apps/files_external/img/app-dark.svg:1]: "Open in new" is well known as a link icon, so not sure.
Maybe also an option:
https://fonts.google.com/icons?icon.query=server&selected=Material+Symbols+Outlined:dns:FILL@1;wght@400;GRAD@0;opsz@20&icon.size=20&icon.color=%23fff

File: apps/files_versions/img/app.svg
Comment (susnux) [reply apps/files_versions/img/app.svg:1]: Makes sense, fixed

File: apps/files_versions/img/app-dark.svg
Comment (susnux) [reply apps/files_versions/img/app-dark.svg:1]: Makes sense, fixed

**Comment by susnux** [comment]
Potential follow up:
There seem to be quite some images we do not use anymore and possibly can remove.
Also the weather status has some other icons that we could migrate to Material Symbols (they provide all kind of weather icons).

File: apps/files_external/img/app-dark.svg
Comment (AndyScherzinger) [reply apps/files_external/img/app-dark.svg:1]: Well, not too important, so I am also fine to just merge this one as is.
If we get feedback that it confuses folks, we can still change it then.
DNS icon also looks fine from my pov.

**Comment by AndyScherzinger** [comment]
Let's merge it than as-is right now, to have it deployed.

Replacing weather icons with material would be nice in case there actually are variants for all cases.

**Review by AndyScherzinger — APPROVED** [review]

**Pull request merged**